src/objects/table/table.scss

```diff
@@ -0,0 +1,84 @@
+@use "../../design-tokens/colors.yml";
+@use "../../design-tokens/font.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/ms";
+
+$table-border-color: colors.$gray-light;
+$table-border-thin: sizes.$edge_small solid $table-border-color;
+$table-border-thick: sizes.$edge_medium solid $table-border-color;
+
+/**
+ * 1. Override some table defaults
+ * 2. Tables may overflow horizontally. Keep this from breaking layouts while
+ *    ensuring content is viewable
+ *    In the future we may want to consider making it visually obvious when
+ *    tables are clipped.
+ * 3. Override default `th` text alignment
+ */
+
+.o-table {
+  border-collapse: collapse; /* 1 */
+  border-spacing: 0; /* 1 */
+  overflow-x: auto; /* 2 */
+  text-align: left; /* 3 */
+  width: 100%; /* 2 */
```

Comment:
Honest question: Would any of these be good defaults for `<table>`?

Reply:
FWIW, tables are one of the things I've always struggled with in design systems. Bootstrap treats `.table` as an opt-in class, but then provides no default table styles, which always felt strange to me. But then I worked at a job where we actually had default table styles, and it was a hassle having to override them with modifier classes for variations.

I could go either way on treating table styles like a component (meaning fully respecting BEM class naming rules, even though I can't think of a situation where I'd use table classes without table markup) or just applying these as the based table styles (meaning don't use class names at all, but instead target the `table`, `th`, and `td` elements).

TL;DR: There's no "right" answer here, unless we know up-front that we'll have lots of table variations.

Reply:
> TL;DR: There's no "right" answer here, unless we know up-front that we'll have lots of table variations.

One benefit of this project is that we have 347 posts and 28 pages of existing content we can reference to see how we're using stuff!

That doesn't mean we should necessarily make decisions based on past behavior that make future changes harder, but it is a nice guiding light.

Reply:
Hmmm, I could see the following rules being useful applied to tables in general, since these kinda feel like they should be the defaults:

```css
table {
   border-collapse: collapse;
   border-spacing: 0;
   width: 100%;
   text-align: left;
}
```

I'd be hesitant to apply the overflow or other rules globally. Do y'all think that makes sense?

Reply:
The rules mentioned above are now applied to `<table>`s globally.